app.js

```diff
@@ -1,97 +1,140 @@
+                </div>
+                <p class="about_person">${author.bio || 'No biography available'}</p>
+            `;
+      container.appendChild(personDiv);
+    });
+  } catch (error) {
+    console.error("Error setting authors:", error);
   }
 }
 
+// set contributors in the DOM
 async function setContributors(apiURL, element) {
-  const contributors = await getAuthors(apiURL);
-  if (contributors.length === 0) {
-    element.innerHTML = "No developers found";
-    return;
-  }
-  
-  // Clear loading message
-  element.innerHTML = "";
-  // List developer names
-  for (const contributor of contributors) {
-    element.innerHTML += `
-      <span class="developer-name">
-        <a href="${contributor.html_url}" class="tooltip" target="_blank">  
-          <img src="${contributor.avatar}" alt="" class="shining-image">  
-          <span class="tooltip-text">${contributor.name}</span>  
-        </a>
-      </span>
-    `;
+  try {
+    const contributors = await getAuthors(apiURL);
+    element.innerHTML = contributors.length === 0 ? "No developers found" :
+      contributors.map(contributor => `
+                <span class="developer-name">
+                    <a href="${contributor.html_url}" class="tooltip" target="_blank">  
+                        <img src="${contributor.avatar}" alt="${contributor.name}'s avatar" class="shining-image">  
+                        <span class="tooltip-text">${contributor.name}</span>  
+                    </a>
+                </span>
+            `).join('');
+  } catch (error) {
+    console.error("Error setting contributors:", error);
+    element.innerHTML = "Error loading developers";
   }
 }
 
+// load all contributors
 async function loadAllContributors() {
   const developerLists = document.querySelectorAll(".developers-list");
-  for (const devList of developerLists) {
-    const apiURL = devList.getAttribute("data-value"); // Get API URL from data-value attribute
-    await setContributors(apiURL, devList); // Fetch and set contributors for each element
-  }
+  const promises = Array.from(developerLists).map(devList => {
+    const apiURL = devList.getAttribute("data-value");
+    return setContributors(apiURL, devList);
+  });
+  await Promise.all(promises);
+}
+
+// Scroll-to-top functionality
+function setupScrollToTop() {
+  const toTopButton = document.getElementById('toTop');
+  if (!toTopButton) return;
+
+  window.addEventListener('scroll', () => {
+    if (window.scrollY > 100) {
+      toTopButton.style.display = 'block';
+    } else {
+      toTopButton.style.display = 'none';
+    }
+  });
+
+  toTopButton.addEventListener('click', (e) => {
+    e.preventDefault();
+    window.scrollTo({
+      top: 0,
+      behavior: 'smooth'
+    });
+  });
 }
 
-window.onload = function () {
+// Wait for DOM content to be loaded before executing scripts
+document.addEventListener('DOMContentLoaded', () => {
   setAuthors(members_url);
```

Comment:
this is a follow-up to [that](https://github.com/move-fast-and-break-things/mfbt.community/pull/59/files?diff=split&w=1#r1797626690) comment above

like this
```suggestion
  setAuthors(members_url).catch(console.error);
```